docs/src/prototypes/chatMessageWithPopover/ChatMessageWithPopover.tsx

```diff
@@ -0,0 +1,56 @@
+import { Chat } from '@stardust-ui/react'
+
+import * as React from 'react'
+import Popover, { handleBlur, handleFocus } from './Popover'
+
+const janeAvatar = {
+  image: 'public/images/avatar/small/ade.jpg',
+  status: { color: 'green', icon: 'check' },
+}
+
+class ChatMessageWithPopover extends React.Component {
+  render() {
+    return (
+      <Chat.Message
+        styles={{
```

Comment:
We should apply `styles` with `Provider`: https://github.com/stardust-ui/react/pull/406#issuecomment-441041119

Reply:
I would rather use just ```style``` for that

Reply:
@sophieH29 
`style` props:
- are just going to be inlined directly to the DOM;
- are impossible to override;
- they do not go through the Fella Renderer;
- they are not reusable through classes so abusing them would create huge styling objects (imagine if you have hundreds of `Chat.Message` components, each with their own ` style` object)
etc.

We should avoid using `style` as much as we can.

Reply:
ok, but please, @Bugaa92, give me an example of a proper way for reaching the same goal.

Reply:
UPD: used Provider, please check the last changes if it's ok